runtime/vm/JFRChunkWriter.cpp

```diff
@@ -647,7 +649,7 @@ VM_JFRChunkWriter::writeJVMInformationEvent()
 	_bufferWriter->writeLEB128(JVMInformationID);
 
 	/* write start time */
-	_bufferWriter->writeLEB128(jvmInfo->jvmStartTime);
+	_bufferWriter->writeLEB128(j9time_current_time_millis());
```

Comment:
why is this being changed?

Reply:
`startTime` is supposed to be the start time of the event rather than the start time of the JVM. This was a mistake in the previous PR.